src/Storage/Storage.jl

```diff
@@ -5,9 +5,9 @@ abstract type AbstractStore end
 
 #Define the interface
 """
-  storagesize(d::AbstractStore)
+  storagesize(d::AbstractStore, p::AbstractString)
```

Comment:
```suggestion
  storagesize(d::AbstractStore, [p::AbstractString])
```

This is optional so we can use square brackets to specify: https://github.com/JuliaIO/Zarr.jl/blob/cbcaeadf9d93ec174b850d90ee6db438d962f140/src/ZArray.jl#L69

Reply:
The path is not currently optional when calling `storagesize(d::AbstractStore, p::AbstractString)`. `ZArray` is not a subtype of `AbstractStore`.